### PR TITLE
add unit test to check module file generated by PythonPackage easyblock

### PR DESCRIPTION
**Pull request by boegel**
test that checks for the bug fixed in #840

### CODE REVIEW

**Comment by boegel** [comment]
@wpoely86: please review?

File: test/easyblocks/module.py [hunk 1]
```diff
@@ -90,6 +90,59 @@ def setUp(self):
...
+
+        self.assertTrue(TMPDIR in app.installdir)
+        self.assertTrue(TMPDIR in app.installdir_mod)
+
+        modtxt = None
+        for cand_mod_filename in ['3.14', '3.14.lua']:
+            full_modpath = os.path.join(app.installdir_mod, 'testpypkg', cand_mod_filename)
+            if os.path.exists(full_modpath):
+                modtxt = read_file(full_modpath)
+                break
```
Comment (wpoely86) [inline test/easyblocks/module.py]: and if both tcl and lua are generated?

Comment (boegel) [inline test/easyblocks/module.py]: that's not supported yet; and we don't really care here, that's not what we care about in this test?

**Comment by boegel** [comment]
Thanks for the review @wpoely86, this is a good base to have some sort of tests (beyond init) for all easyblocks...

**Pull request merged**